### PR TITLE
Teensy loader

### PR DESCRIPTION
**Pull request by Baycken**
Teensy Command Line Uploader
for more information, please read [REAMME.md](https://github.com/mcgill-robotics/compsys/blob/teensy_loader/tools/tclu/README.md).

### CODE REVIEW

**Comment by anassinator** [comment]
I wanna discuss this before we merge

File: tools/tclu/Makefile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CC ?= gcc
+CFLAGS ?= -O2 -Wall
+LIB ?= -lusb -ludev -lsysfs
+tclu: tclu.c
+	$(CC) $(CFLAGS) -s -DUSE_LIBUSB -o tclu tclu.c $(LIB)
+
+clean:
+	rm -f tclu
```
Comment (dlavoieb) [inline tools/tclu/Makefile]: Is there an install target?

**Comment by dlavoieb** [comment]
Should the install script build and install (or at least add the build dir to path) the tool directly.

**Comment by anassinator** [comment]
I believe this is worthy of being a separate repository, probably as a fork of the original. CompSys can still install it though.

**Comment by Baycken** [comment]
@dlavoieb There is currently no install target, I could add it, which is just simply `cp` the compiled file to `/usr/local/bin/`. I don't think we should compile this with the install script, as it has dependencies that is not used for anything else and we have to install them with 'apt-get'.
@anassinator We could fork it, and have the install script `wget` the compiled version from github. We can discuss this on Saturday.

**Pull request closed without merging**